src/scenario.py

```diff
@@ -607,19 +607,19 @@ def run(self, strategy_name, options):
             ax.plot(xlabels, list([sum(cs) for cs in sum_cs]), label="CS")
             for name, values in loads.items():
                 ax.plot(xlabels, values, label=name)
-            # draw schedule
-            for gcID, schedule in gcWindowSchedule.items():
-                if all(s is not None for s in schedule):
-                    # schedule exists
-                    window_values = [v * int(max(totalLoad)) for v in schedule]
-                    ax.plot(xlabels, window_values, label="window {}".format(gcID), linestyle='--')
-
-            for gcID, schedule in gcPowerSchedule.items():
-                if any(s is not None for s in schedule):
-                    # schedule exists
-                    ax.plot(xlabels, schedule, label="Schedule {}".format(gcID))
-
-            ax.plot(xlabels, totalLoad, label="total")
+
+            # draw schedule or charge-windows
+            if type(strat).__name__ == "FlexWindow":
```

Comment:
You can use `strategy_name` instead (function argument of Scenario.run). Example:
`if strategy_name == "flex_window":`